js/miniautures.js

```diff
@@ -0,0 +1,28 @@
+import  {urlForExport, likesForExport, commentsForExport} from './data.js';
+
+const pictures = document.querySelector ('.pictures');
+const pictureTemplate = document.querySelector ('#picture')
+  .content
+  .querySelector('a');
+
+const drawPicture = function () {
+  const newPicture = pictureTemplate.cloneNode(true);
+
+  const pictureSrcElement = newPicture.querySelector('src');
+  pictureSrcElement.textContent = urlForExport;
+  const pictureLikesElement = newPicture.querySelector('.picture__likes');
+  pictureLikesElement.textContent = likesForExport;
```

Comment:
Текстовое содержимое элемента с количеством лайков устанавливаем как количество лайков нашей картинки, которую получили в качестве аргумента функции
```suggestion
pictureLikesElement.textContent = picture.likes;
```